src/components/code_editor/code_editor.js

```diff
@@ -159,11 +160,19 @@ export class EuiCodeEditor extends Component {
         data-test-subj="codeEditorHint"
       >
         <p className="euiText">
-          Press Enter to start {activity}.
+          <EuiI18n
+            token={`euiCodeEditor.start${activityLabel}`}
+            default={`Press Enter to start ${activity}.`}
+            values={{ activity }}
+          />
         </p>
 
         <p className="euiText">
-          When you&rsquo;re done, press Escape to stop {activity}.
+          <EuiI18n
+            token={`euiCodeEditor.stop${activityLabel}`}
+            default={` When you're done, press Escape to stop ${activity}.`}
```

Comment:
In case whitespace matters:

```suggestion
            default={`When you're done, press Escape to stop ${activity}.`}
```